packages/bento-design-system/src/Tabs/createTabs.tsx

```diff
@@ -60,9 +71,12 @@ export function createTabs(config: TabsConfig) {
           {icon && (
             <Column width="content">{icon({ size: config.iconSize, color: "inherit" })}</Column>
           )}
-          <Label size={config.labelSize[size]} uppercase={config.uppercaseLabel}>
-            {label}
-          </Label>
+          <Column width="content">
```

Comment:
having `width="content"` on the label or on the rightAccessory doesn't make any difference in this case, right?